WalletWasabi.Documentation/Guides/WasabiCodingGuidelines.md

```diff
@@ -0,0 +1,188 @@
+
+## Example ViewModel
+
+```c#
+  public class MyViewModel : ReactiveObject, IDisposable
+	{
+		private CompositeDisposable _disposables = new CompositeDisposable();
+		private string _myProperty;
+		private bool _myConfirmed;
+		private ObservableAsPropertyHelper<bool> _confirmed;
+		private SmartCoin Model { get; }
+
+		public string MyProperty
+		{
+			get => _myProperty;
+			set => this.RaiseAndSetIfChanged(ref _myProperty, value);
+		}
+
+		public bool MyConfirmed
+		{
+			get => _myConfirmed;
+			set => this.RaiseAndSetIfChanged(ref _myConfirmed, value);
+		}
+
+		public ReactiveCommand MyCommand { get; }
+		public ReactiveCommand MySecondCommand { get; }
+
+		public MyViewModel()
+		{
+			//need to detect propertyChance at class member https://reactiveui.net/docs/guidelines/framework/dispose-your-subscriptions
```

Comment:
Comments are full sentences with proper punctuation.  
There should be also spacing: 

- BAD: `//I'm a comment.`
- GOOD: `// I'm a comment.`

Reply:
- VERY BAD: `//imacomment

Reply:
ack!